apps/src/blockly/utils.ts

```diff
@@ -16,19 +21,19 @@ export const TRUTHY_DEFAULT = attributeValue => attributeValue !== 'false';
  * @returns {boolean} The boolean value of the attribute as determined by the callback function.
  */
 export function readBooleanAttribute(
-  xmlElement,
-  attribute,
+  xmlElement: Element,
+  attribute: string,
```

Comment:
For my own TS knowledge/outside the scope of this PR: Is there any value added by defining this as an ENUM or something so we're explicit about the attributes we could be reading/avoiding typos? IIRC, I defined one-off constants like `USER_CREATED_XML_ATTRIBUTE,` but I wonder if TS could make this cleaner/more succinct.

Reply:
good question, do we know that these will always be specific values? If we do, we can definitely specify the expected values as an enum or type.

Reply:
Yes and no is the answer to your question. We could start reading a new XML attribute, but then we'd probably want to use that string in many different places and not risk misspelling it anywhere. So, at any given moment, we should be able to deduce that we're only reading specific values, but that list is not static.